patches/update-cache-path.patch

```diff
@@ -7,7 +7,7 @@ index 93424ca..133657e 100644
  	@memoize
  	get cachePath(): Promise<string> {
 -		const result = path.join(tmpdir(), `vscode-update-${this.productService.target}-${process.arch}`);
-+		const result = path.join(tmpdir(), `${this.productService.nameShort.toLowerCase()}-update-${this.productService.target}-${process.arch}`);
++		const result = path.join(tmpdir(), `${this.productService.applicationName}-update-${this.productService.target}-${process.arch}`);
```

Comment:
I think we'd still want to keep the `toLowerCase` though, for consistency sake

Reply:
I think `this.productService.applicationName` is fine as it is since it's also the name of the executable (without any lower casing).
But yeah, the code is better than the one I did...